css-brochure-form/index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div>
+    <h1>LEARN MORE</h1>
+  </div>
+  <form>
+    <fieldset>
+     <div>
+       <div class="form-headers">
+        <label for="company-email">COMPANY EMAIL</label>
+        <br />
```

Comment:
Avoid using `<br>` tags, if you want to create some spacing add some bottom margin to your label elements